src/service/translate.js

```diff
@@ -1714,10 +1714,8 @@ function $translate($STORAGE_KEY, $windowProvider) {
           }
 
           var allTranslationsLoaded = function (tableData) {
+            $translationTable = {};
             angular.forEach(tableData, function (data) {
-              if ($translationTable[data.key]) {
-                delete $translationTable[data.key];
-              }
```

Comment:
To keep things linked together.
I have just explained why I think that this is a bugfix [here](https://github.com/angular-translate/angular-translate/issues/988#issuecomment-94680684).